app/routers/v1/scraper.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import APIRouter, HTTPException, Depends
+from pydantic import BaseModel
+from app.services.github_jobs_scraper import GitHubJobsScraper
+from app.dependencies.auth import validate_api_key
+
+router = APIRouter()
```

Comment:
router = APIRouter(
    prefix="/scrape",
    responses={404: {"description": "Not found"}},
)

Can you add a prefix instead and change the file name to `scrape.py`?